tailcfg/tailcfg.go

```diff
@@ -2487,6 +2488,11 @@ const (
 	// If multiple values of this key exist, they should be merged in sequence
 	// (replace conflicting keys).
 	NodeAttrServiceHost NodeCapability = "service-host"
+
+	// NodeAttrMaxKeyDuration represents the MaxKeyDuration setting on the
+	// tailnet. The value of this key in [NodeCapMap] will be only one entry of
+	// type [Duration]: an int64 representing the time in nanoseconds.
+	NodeAttrMaxKeyDuration NodeCapability = "max-key-duration"
```

Comment:
How does this differ from `MapResponse.MaxKeyDuration` from capability 85?

Reply:
We never shipped the server-side of that, and based on input from Brad, I'm putting this in the CapMap instead of at the top-level MapResponse. I intend to remove the code to handle it in MapResponse, which shouldn't affect anything.

Reply:
Maybe "tailnet.maxKeyDuration" so we have the `tailnet.` prefix available to advertise additional properties of the tailnet config to the self node attrs in the future?